src/components/OverlayConfigDialog.js

```
@@ -0,0 +1,207 @@
+      delete base.dy;
+      delete base.dx;
+    }
+
+    mapping[this.field][this.index] = base;
+    store.set('mapping', mapping);
+    this.destroy();
+  }
+
+  destroy() {
+    if (this.overlay && this.overlay.parentNode) {
+      this.overlay.parentNode.removeChild(this.overlay);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)